docs/pair_hmm.tex

```diff
@@ -0,0 +1,109 @@
+\begin{itemize}
+\item $(i, j, M/D/I) \rightarrow (i+1, j+1, M)$: match of read position $i+1$ with haplotype position $j + 1$
+\item $(i, j, M/D) \rightarrow (i, j+1, D)$: deletion after read position $i$ -- haplotype position $j+1$ is deleted
+\item $(i, j, M/I) \rightarrow (i + 1, j, I)$: insertion at read position $i + 1$
+\end{itemize}
+Note that the state label $s$ seems redundant because it can be reconstructed from the sequence of $i$ an $j$.  While this is true, our model treats indel starts differently from indel continuations, and by including the state label we can distinguish these conveniently\footnote{That is, we can tell which type of transition it is by looking back one unit instead of two.  Thus we have a first-order Markov model instead of a second-order Markov model.}.  These transitions are illustrated as a finite state machine in Figure \ref{fig:fsm}.
+
+\begin{figure}
+\center
+\includegraphics[scale=0.35]{finite_state.png}
+\caption{Finite state machine of transitions among match, insertion, and deletion alignment states.}
+\label{fig:fsm}
+\end{figure}
+
+The read-alignment likelihood has two components.  First is the probability of the sequence of match, insertion, and deletion states, which is
+\begin{equation}
+P(\mc{A}) = \prod_k T_{s_k, s_{k+1}},
+\end{equation}
+where $T$ is a matrix of state transition probabilities\footnote{The gap continuation probability corresponds to a phred-scaled quality that is set by the \code{gcpHMM} argument, which is 10 by default, implying that $T_{DD} = T_{II} = 10^{-10/10} = 1/10$ and $T_{DM} = T_{IM} = 1 - T_{II}$.  The indel start transitions are derived from the read's BQSR base insertion and base deletion qualities, if they exist, or a phred-scaled indel start quality of 45 if they do not.  In production at the Broad Institute bams do not have BQSR indel qualities, hence the constant default is used.  That is, $T_{MD} = T_{MI} = 10^{-4.5}$ and $T_{MM} = 1 - 2 \times 10^{-4.5}$.}.  The index $k$ runs from 1 to the number of states in the alignment, that is, the read length plus the number of deleted reference bases.  Next is the emission probability of the read bases given the haplotype bases they align to and the base qualities:
+\begin{equation}
```

Comment:
The following legend can accompany the state transition diagram:
![image](https://user-images.githubusercontent.com/8916428/29697201-b9caf2f0-891b-11e7-91cf-954883fc323a.png)
taken from https://docs.google.com/presentation/d/1hCxMjf1VF8eJRiKt5wQQAehhMkjUS-h1n5Ubr68nRGE/edit#slide=id.p30

Reply:
I now include this information in the figure caption.